crates/revm/src/db/states/bundle_state.rs

```diff
@@ -692,12 +691,12 @@ mod tests {
         );
     }
 
-    const fn account1() -> B160 {
-        B160([0x60; 20])
+    const fn account1() -> Address {
+        Address(FixedBytes([0x60; 20]))
```

Comment:
`Address::new(..)` maybe if it is const?

Reply:
https://github.com/bluealloy/revm/pull/724/commits/42866111e858f2f3a2710563045b61b7779eff88